clang/lib/CIR/CodeGen/TargetInfo.h

```diff
@@ -62,6 +62,13 @@ class TargetCIRGenInfo {
                            std::vector<LValue> &ResultRegDests,
                            std::string &AsmString, unsigned NumOutputs) const {}
 
+  /// Get target favored AST address space of a global variable for languages
```

Comment:
*`flavored`?

Reply:
This comment is copied from original method ; ) I think "favor" here means more or less "prefer".